interface/super/edit_layout.php

```diff
@@ -211,31 +211,30 @@ function swapGroups($id1, $id2)
     }
 }
 
+function tableNameFromLayout($layout_id)
+{
+    // Skip layouts that store data in vertical tables.
+    if (substr($layout_id,0,3) == 'LBF' || substr($layout_id,0,3) == 'LBT' || $layout_id == "FACUSR") {
+        return '';
+    }
+    if      ($layout_id == "DEM") $tablename = "patient_data";
+    else if ($layout_id == "HIS") $tablename = "history_data";
+    else if ($layout_id == "SRH") $tablename = "lists_ippf_srh";
+    else if ($layout_id == "CON") $tablename = "lists_ippf_con";
+    else if ($layout_id == "GCA") $tablename = "lists_ippf_gcac";
```

Comment:
Looking for input here. Travis CI complains about the 5 lines above but I think the format makes the code easier to read and easier to spot errors. Are we being too pedantic?

Reply:
hi @sunsetsystems ,
We converted to PSR2 standards awhile back. There were issues brought up (similar to yours), but the decision was to proceed (ie. the pros outweighed the cons). A lot of work was put into the PSR2 code conversion and travis by Robert and Jason, and would not be comfortable allowing any code into the codebase that breaks travis. Once your code editor is set up to follow PSR2, it becomes much easier to deal with. Here's the PSR2 section on if/else:
http://www.php-fig.org/psr/psr-2/#51-if-elseif-else
(although unrelated, in case you are wondering, PSR2 does allow ternary conditionals)